pkg/pluginregistry/bundles.go

```diff
@@ -25,7 +25,7 @@ func (r *PluginRegistry) NewTestStepBundle(testStepDescriptor test.TestStepDescr
 	}
 	label := testStepDescriptor.Label
 	if label == "" {
-		label = testStepDescriptor.Name
+		return nil, ErrStepLabelIsMandatory{testStepDescriptor}
```

Comment:
Struct initialization without explicit field name should be avoided - I thought we had a linter check in place for that

Reply:
Fixed.